pkg/network/protocols/http/telemetry.go

```diff
@@ -8,15 +8,13 @@
 package http
 
 import (
-	"time"
-
 	libtelemetry "github.com/DataDog/datadog-agent/pkg/network/protocols/telemetry"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
-	"go.uber.org/atomic"
 )
 
 type Telemetry struct {
-	LastCheck                                   *atomic.Int64
+	metricGroup *libtelemetry.MetricGroup
```

Comment:
can you document? metricGroup does not explain its purpose

Reply:
There is documentation here explaining what a `MetricGroup` does:
https://github.com/DataDog/datadog-agent/blob/pedro/telemetry-changes/pkg/network/protocols/telemetry/metric_group.go
Please let me know if that clarifies things.